build/scripts/meta.yaml.schema

```diff
@@ -34,7 +34,7 @@
     },
     "icon": {
       "type": "string",
-      "pattern": "[-._a-zA-Z0-9]+.svg[^.]*$"
+      "pattern": "[-._a-zA-Z0-9]+?\\.(?:svg|png)(?:\\?.*)?$"
```

Comment:
I don't think this regex works, I tested it against `file.png` and `file.svg` and it didn't match.

Reply:
@vitaliy-guliy quite difficult regexp, could you elaborate about each of the segments?

Like

 ^[-\\._a-zA-Z0-9]+\\.(svg)|(png)$
 
 ^ **->** the beging of the line
 [-\\._a-zA-Z0-9] **->** just file name that could contain -._letters digits
 \\.(svg)|(png) **->** file extension must be .svg or .png
 $ **->** the end

Reply:
@akurinnoy helped me to write this regexp, as he mush more experienced.
We have to use double slash when using a regexp as a value in json.
The original regexp is `[-._a-zA-Z0-9]+?\.(?:svg|png)(?:\?.*)?$`. 
Where:
- `[-._a-zA-Z0-9]+?` checks for the file name
- `\.(?:svg|png)` the extension, which can be only .svg or .png
- `(?:\?.*)?` optional query string

Reply:
So, query param... It's not really file path but URL?
Then `[-._a-zA-Z0-9]+` does not really make sense... It may work since `.` is not escaped and it includes any character... Or `^` is missing by purpose...
OK, let's discuss offline

Reply:
We don't need ^ at the start of the regexp, isn't it?

Reply:
You can check the regexp online https://regexr.com/50au9

Reply:
> So, query param... It's not really file path but URL?

It looks like it could be a path also. If it's a path, it points on an image on the current plugin registry. So, both variants.

Reply:
Oleksii explained to me. 

> I don't think this regex works, I tested it against file.png and file.svg and it didn't match.
@ericwill How did you test? If with some online tool - note that you must replace \\ with \

It works fine for me 
![Screenshot_20200313_125417](https://user-images.githubusercontent.com/5887312/76615034-dfbd2180-6529-11ea-9bc9-f0495a6bbaf6.png)

1. png and svg are allowed
2. .ico or references without file extension are not.
3. Query params are allowed

The only comment that since we miss `^` - user could provide non-valid URL at all, like `$*WUE*#@$&@*#ico.png` and it would be not caught by our check. But it's not issue that is caused by your changes.

Reply:
@vitaliy-guliy Please update https://github.com/eclipse/che-plugin-registry#plugin-meta-yaml-structure and add png there as well.